swagger_server/__main__.py

```diff
@@ -137,10 +129,10 @@ def start_consumer(thread_queue, db_instance):
                 else:
                     logger.info("got message from MQ: " + str(msg))
             else:
-                db_instance.add_key_value_pair_to_db(MESSAGE_ID, msg)
+                db_instance.add_key_value_pair_to_db(str(MESSAGE_ID), msg)
```

Comment:
It seems that this `str()` is redundant since there's another `str()` in `add_key_value_pair_to_db()`?